### PR TITLE
Geographical migration #94 (BROKEN, DO NOT MERGE)

### LINKED ISSUE #94: NTR: Geographical migration
For new term requests, please provide the following information:

## Preferred term label

seasonal migration

## Textual definition

a rhythmic behavior of an organism that involves regular movement from one geographic area to another on a yearly cycle. I'm not sure if the "migratory" nature is important here. Is it important to capture that many individuals are moving together?

## Suggested parent term

Rhythmic behavior

## Attribution

http://orcid.org/0000-0002-2908-3327

### PR DESCRIPTION
**Pull request by DitchingIt**
Fixes #94

### CODE REVIEW

**Comment by DitchingIt** [comment]
Fixes #94

File: src/ontology/nbo-edit.owl [hunk 1]
```diff
@@ -9938,6 +9945,37 @@ AnnotationAssertion(oboInOwl:hasOBONamespace obo:NBO_0050002 "behavior_ontology"
 AnnotationAssertion(rdfs:label obo:NBO_0050002 "circling behavior")
 SubClassOf(obo:NBO_0050002 obo:NBO_0000013)
 
+# Class: obo:NBO_0070001 (migration)
+
+AnnotationAssertion(obo:IAO_0000115 obo:NBO_0070001 "Rhythmic behavior that involves cue-initiated, persistent, direct, focused movement between habitats.")
```
Comment (matentzn) [inline src/ontology/nbo-edit.owl:9950]: Missing source

**Review by matentzn — CHANGES_REQUESTED** [review]

File: src/ontology/nbo-edit.owl [hunk 2]
```diff
@@ -9949,6 +9954,37 @@ AnnotationAssertion(oboInOwl:hasOBONamespace obo:NBO_0050002 "behavior_ontology"
...
+AnnotationAssertion(terms:date obo:NBO_0070002 "2023-04-22T14:41:53Z"^^xsd:dateTime)
+AnnotationAssertion(oboInOwl:source obo:NBO_0070002 "Bandara K, Varpe Ø, Wijewardene L, Tverberg V, Eiane K. Two hundred years of zooplankton vertical migration research. Biological Reviews. 2021 Aug;96(4):1547-89.")
+AnnotationAssertion(rdfs:label obo:NBO_0070002 "diel vertical migration")
+SubClassOf(obo:NBO_0070002 obo:NBO_0070001)
+
+# Class: obo:NBO_0070003 (geographical migration)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1146/annurev-ecolsys-012021-031035") obo:IAO_0000115 obo:NBO_0070003 "Migration that involves regular movement laterally between distant ranges in seasonal cycles.")
+AnnotationAssertion(dc:contributor obo:NBO_0070003 <http://orcid.org/0000-0002-2908-3327>)
+AnnotationAssertion(terms:contributor obo:NBO_0070003 <https://orcid.org/0000-0002-4277-2151>)
```
Comment (matentzn) [inline src/ontology/nbo-edit.owl:9981]: You sometimes use `terms:contributor` and other times `dc:contributor` - you should only ever use the former.

**Comment by matentzn** [comment]
This PR is broken now - you should redo it from scratch; its conflicting with all the other PRs you did previously.

**Pull request closed without merging**